src/main/scala/io/radanalytics/streaming/amqp/AMQPFlowController.scala

```diff
@@ -15,22 +15,21 @@
  * limitations under the License.
  */
 
-package org.apache.spark.streaming.amqp
+package io.radanalytics.streaming.amqp
 
 import java.lang.Long
 import java.util.concurrent._
 
 import io.vertx.core.{Handler, Vertx}
 import io.vertx.proton.{ProtonDelivery, ProtonMessageHandler, ProtonReceiver}
 import org.apache.qpid.proton.message.Message
-import org.apache.spark.internal.Logging
+import org.slf4j.LoggerFactory
 
 import scala.collection.mutable
 
 /**
   * Listener for events from an AMQP flow controller
   */
-private [streaming]
```

Comment:
Why not having this as `private` in the `streaming` package ? (Note : this comment is valid for all the other removed `private[streaming]`)

Reply:
Because this class has a different package name (`org.apache.spark.streaming.amqp`) than the classes that uses this class in the repo (`io.radanalytics..`). It's related to https://github.com/radanalyticsio/streaming-amqp/pull/25#discussion_r154333612


If everything is in the same package then yes, those `private [streaming]` could have stayed.

Reply:
As far as I can see this class is in the same package (line 18). What am I missing ?

Reply:
you are right, this one is not necessary. I am removing it.

Reply:
You are right ! Because this references and need for using Spark internal stuff, this is way most of the connectors live in the `org.apache.spark.streaming` package I guess.